starter-code/index.html

```diff
@@ -0,0 +1,246 @@
+        <ol class="carousel-indicators">
+          <li data-target="#carouselExampleCaptions" data-slide-to="0" class="active"></li>
+          <li data-target="#carouselExampleCaptions" data-slide-to="1"></li>
+          <li data-target="#carouselExampleCaptions" data-slide-to="2"></li>
+        </ol>
+        <div class="carousel-inner">
+          <div class="carousel-item active">
+            <img src="./img/slider-1.jpg" class="d-block w-100 myGrayOverlay" alt="...">
+            <div class="carousel-caption d-none d-md-block">
+              <h1>ETIAM MAURIS TORTOR, PHARETRA QUIS LOBORTIS IN, PHARETRA IN DIAM</h1>
+              <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+              <button type="button" class="btn btn-outline-light">READ MORE</button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img src="./img/slider-2.jpg" class="d-block w-100 myGrayOverlay" alt="...">
+            <div class="carousel-caption d-none d-md-block">
+              <h1>FALAURIS SOLLICITUDIN QUIS DOLOR VENENATIS FACILISIS</h1>
+              <p>Donec sed dolor eu augue dignissim pellentesque. Fusce semper tortor ornare, luctus sem in, dictum leo. Ut at congue quam! Maecenas luctus interdum odio, eget volutpat velit hendrerit eu? Suspendisse at magna ut sem blandit hendrerit non ut magna. Cras quis porta dolor. Aliquam erat volutpat. Phasellus fringilla feugiat libero nec volutpat. Maecenas fermentum bibendum […]</p>
+              <button type="button" class="btn btn-outline-light">READ MORE</button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img src="./img/slider-3.jpg" class="d-block w-100 myGrayOverlay" alt="...">
+            <div class="carousel-caption d-none d-md-block">
+              <h1>SUSPENDISSE ORNARE, FELIS FEUGIAT SUSCIPIT PHARETRA</h1>
+              <p>Quisque sit amet sagittis justo. Suspendisse ornare, felis feugiat suscipit pharetra, lorem est viverra odio; tristique tristique diam tellus id tellus. In consectetur aliquet tristique. Curabitur imperdiet leo in nisl consectetur mattis. Quisque bibendum, mauris vitae suscipit ornare; est nibh elementum mauris, eu malesuada nunc magna at justo. Proin blandit id tellus non pellentesque. Mauris […]</p>
+              <button type="button" class="btn btn-outline-light">READ MORE</button>
+            </div>
+          </div>
+        </div>
+        <a class="carousel-control-prev" href="#carouselExampleCaptions" role="button" data-slide="prev">
+          <span class="carousel-control-prev-icon" aria-hidden="true"></span>
```

Comment:
You can add class ".bg-danger" so it'll match the theme even more. So <span class="carousel-control-prev-icon bg-danger" aria-hidden="true"></span>